accounts/forms.py

```diff
@@ -327,3 +319,18 @@ def __init__(self, *args, **kwargs):
                 }
         super(DeleteUserForm, self).__init__(*args, **kwargs)
 
+
+class EmailSettingsForm(forms.Form):
+    enabled_stream_emails = forms.BooleanField(
```

Comment:
The default value for stream email is False so that means we will start with a lot of entries in the UserEmailSettings table. given that we have other fields regarding stream email I thought we could leave this separate, but if you want I can change it.